common/lib/xmodule/xmodule/js/fixtures/video_all.html

```diff
@@ -71,16 +71,23 @@
 
         <ul class="wrapper-downloads">
           <li class="video-tracks">
-            <div class="a11y-menu-container">
-                <a class="a11y-menu-button" href="#" title=".srt">.srt</a>
-                <ol class="a11y-menu-list">
-                  <li class="a11y-menu-item">
-                    <a class="a11y-menu-item-link" href="#txt" title="Text (.txt) file" data-value="txt">Text (.txt) file</a>
-                  </li>
-                  <li class="a11y-menu-item active">
-                    <a class="a11y-menu-item-link" href="#srt" title="SubRip (.srt) file" data-value="srt">SubRip (.srt) file</a>
-                  </li>
-                </ol>
+            <div class="wrapper-more-actions">
+              <button type="button" class="button button-more has-dropdown" aria-haspopup="true" aria-expanded="false" aria-controls="dropdown-transcript-format" aria-disabled="false">
+                <span class="icon fa fa-sort-desc" aria-hidden="true"></span>
```

Comment:
@talbs Just talked to @cptvitamin and these items are okay as-is. Users will understand the controls without more specificity.